ztp/gitops-subscriptions/argocd/example/image-based-upgrades/README.md

```diff
@@ -0,0 +1,91 @@
+
+Ensure that your Git repository, which will be used with the ArgoCD policies application, contains a directory structured as follows:
+
+```plaintext
+├── source-crs/
+│   ├── ibu/
+│   │    ├── PlatformBackupRestore.yaml
+├── ...
+├── custom-oadp-workload-crs.yaml
+├── example-oadp-policy.yaml
+├── kustomization.yaml
+```
+
+Note that [`source-crs/ibu/PlatformBackupRestore.yaml`](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/ibu/PlatformBackupRestore.yaml) is provided in the ZTP image, however, it is important to ensure that the `kustomization.yaml` file is located in the same directory structure shown above in order to reference the `PlatformBackupRestore.yaml` manifest.
+
+### Generating the ConfigMap and Policy
+
+Use the [`configMapGenerator`](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#configmapgenerator) provided by the Kustomize tool to generate a ConfigMap encapsulating the OADP CRs for IBU. The OpenShift platform-related CRs are available [here](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/ibu/PlatformBackupRestore.yaml).
+
+Additional workload OADP CRs may be included in the ConfigMap generation process as shown in the example below:
+
+```yaml
+configMapGenerator:
+- files:
+  - source-crs/ibu/PlatformBackupRestore.yaml
+  - custom-oadp-workload-crs.yaml
+  name: oadp-cm
+  namespace: ztp-common
+
+generatorOptions:
+  disableNameSuffixHash: true
+```
+
+Ensure that the `custom-oadp-workload-crs.yaml` file includes a one-to-one mapping of OADP backup and restore CRs. It's important to note that these CRs can be stored either in separate YAML manifests or consolidated within a single YAML file (as shown below), with each CR section separated by the `---` directive.
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: foobar-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - foobar
+  includedNamespaceScopedResources:
+  - secrets
+  - deployments
+  - statefulsets
+  excludedClusterScopedResources:
+  - persistentVolumes
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: foobar-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3"
+spec:
+  backupName:
+    foobar-app
+```
+
+> [!IMPORTANT]
+> Don't forget to create and include the policy [example-oadp-policy.yaml](./example-oadp-policy.yaml) under the `resources` object in the `kustomization.yaml` file.
+> The policy uses hub-side templating and requires Advanced Cluster Management (ACM) 2.9 and later versions.
```

Comment:
I think the copyConfigMapData function exists in ACM 2.8 as well.

Reply:
Changed to 2.8 as per your comment and [release note](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html-single/release_notes/index). Thanks!

Reply:
ACM 2.9.2 will be the minimum support acm version for IBU

Reply:
@browsell - should we set the version dependency here as well (i.e. instead of 2.8 we explicitly state that IBU requires ACM 2.9.2+).

Reply:
On second thought. I can state that ACM 2.8+ is required for hub side templating, however, ACM 2.9.2 will be the minimum support acm version for IBU. This statement is technically sound and also addresses both comments.